databases/week1/db1-homework-my_squirrel.sql

```diff
@@ -0,0 +1,85 @@
+-- Hello My_SQuirreLs, here is my homework ---> databases-1 (a.k.a. week 15) ---> Magda /HYF#12
+
+USE hyf_db_1;   -- all the comments/questions will be removed once resolved :) 
+SELECT * FROM status; 
+SELECT * FROM task;  --  jest powiązany z tabelą status przez status_id i z tabelą user przez user_id
+SELECT * FROM user;   
+
+-- 01 -- Find out how many tasks are in the task table: 
+SELECT count(id) AS `number_of_tasks_in_task_table`   -- backticks work with column aliases but not the values!!!
+FROM task;
+-- Question: Is the alias too long? Or should I leave it as descriptive as possible?
+-- Question: Would count(*) be too redundant? 
+
+-- 02 -- Find out how many tasks in the task table do not have a valid due date: 
+SELECT count(id) AS `tasks_in_task_table_without_valid_due_date`  
+FROM task 
+WHERE due_date IS NULL;  
+
+-- 03 -- Find all the tasks that are marked as done: 
+SELECT task.*, status.name
+FROM task INNER JOIN status 
+ON task.status_id = status.id 
+WHERE status.name = 'Done';  -- [LIKE] equals [=] 
+-- or --
+SELECT task.*, status.id
+FROM task INNER JOIN status 
+ON task.status_id = status.id 
+WHERE status.id = 3; 
```

Comment:
do you really need the join when using this option? ;p

Reply:
```
SELECT title AS `done_tasks_title`, description AS `done_tasks_description`, status_id AS `done_tasks_id`
FROM task 
WHERE status_id = 3;
``` 
Do you mean sth like this?

Reply:
yes, that should work just as well :)

Reply:
Thank you :)